components/x-eventpromo/src/lib/animation-control.js

```diff
@@ -0,0 +1,18 @@
+module.exports = () => {
+	const aniCntr = document.querySelector('.js-event-promo__control');
+	const aniEls = document.querySelectorAll('.js-event-promo--animate');
+
+	if (!aniEls || !aniCntr) {
+		return;
+	}
+
+	aniCntr.addEventListener('click', () => {
+		aniEls.forEach((item) => {
+			item.classList.toggle('paused');
+		});
+
+		aniCntr.classList.toggle('event-promo__control--paused');
+	});
+
+	return true;
+};
```

Comment:
What's this file for?

Reply:
a mistake :( 
That a copy of /lib file from n-eventpromo, not used here... I'll delete them.